.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,41 +2,46 @@
 
 # Checklist:
 
-- I have performed a self-review of my own code
-- I have commented my code, particularly in hard-to-understand areas
-- I have made corresponding changes to the documentation
-- I have added tests that prove my fix is effective or that my feature works
-- New and existing unit tests pass locally with my changes
+- I have performed a self-review of my own code,
+- I have commented my code, particularly in hard-to-understand areas,
+- I have made corresponding changes to the documentation,
+- I have added tests that prove my fix is effective or that my feature works,
+- New and existing unit tests pass locally with my changes ("bundle exec rake"),
+- Title include "WIP" if work is in progress.
+
 -->
 
-# Resolves #001 <!--fill issue number-->
+Resolves #001 <!--fill issue number-->
+
+### Description
+<!-- Please include a summary of the change and which issue is fixed. 
+Please also include relevant motivation and context.
+Guide questions:
+  - What motivated this change (if not already described in an issue)?
+  - What alternative solutions did you consider?
+  - What are the tradeoffs for your solution?
+   
+List any dependencies that are required for this change. (gems, js libraries, etc.)
 
-# Description
-<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
-List any dependencies that are required for this change. Put "WIP" in the title if work is in progress. Did you add any dependencies?
 Include anything else we should know about. -->
 
-## Type of change
+### Type of change
 
 <!-- Please delete options that are not relevant. -->
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] This change requires a documentation update
-- [ ] Documentation update
+* Bug fix (non-breaking change which fixes an issue)
+* New feature (non-breaking change which adds functionality)
+* Breaking change (fix or feature that would cause existing functionality to not work as expected)
+* This change requires a documentation update
+* Documentation update
 
-# How Has This Been Tested?
-<!--Optional. Delete if not relevant-->
+### How Has This Been Tested?
 
 <!-- Please describe the tests that you ran to verify your changes. 
 Provide instructions so we can reproduce. 
-Please also list any relevant details for your test configuration -->
-
-- [ ] Test A
-- [ ] Test B
+Do we need to do anything else to verify your changes? 
+If so, provide instructions (including any relevant configuration) so that we can reproduce? -->
 
+### Screenshots
 
-# Screenshots
-<!--Optional. Delete if not relevant. 
```

Comment:
It looks like this line was deleted by accident.

Reply:
Oh yeah, true.